### PR TITLE
Buzz Tab - Task

### PR DESCRIPTION
**Pull request by Mohammad-Abohasan**
## Buzz - The user should be able to add a post

**Given** The user logged in to the system,
**And** The user opens the Buzz tab.
 
**When** The user types post content (writes a file and stores it in fixtures, then reads from it),
**And** The user clicks the Post button.

**Then** The user's post should exist in the page.

### CODE REVIEW

**Comment by Mohammad-Abohasan** [comment]
## [Test-Replay](https://cloud.cypress.io/projects/hef2z5/runs/60/overview/c4861eb8-78b7-40ff-9e62-67053759bd72/replay)
[BuzzTab-Task](https://cloud.cypress.io/projects/hef2z5/runs)

File: cypress/support/pageObjects/buzzTab/BuzzTabActions.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    SharedHelper.mainMenuItems().contains("Buzz").click();
+  }
+
+  addNewPost(postData: string) {
+    this.elements.postInputField().type(postData);
+    this.elements.postButton().click();
+    SharedHelper.checkToastIsExist(false);
+  }
+
+  clearBuzzTabDirectoryFromFixtures() {
```
Comment (Mohammad-Abohasan) [inline cypress/support/pageObjects/buzzTab/BuzzTabActions.ts:19]: delete from POM

File: cypress/e2e/buzzTab/buzzTab.spec.cy.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  it("Buzz - The user should be able to add a post", () => {
+    cy.fixture("buzzTab/buzz.json").then((postData) => {
+      buzzTabActions.addNewPost(postData.post);
+      buzzTabAssertions.checkPostText(postData.post);
+    });
+  });
+
+  afterEach(() => {
+    buzzTabActions.clearBuzzTabDirectoryFromFixtures();
```
Comment (Mohammad-Abohasan) [inline cypress/e2e/buzzTab/buzzTab.spec.cy.ts:24]:
```suggestion
    cy.task("deleteFile", "cypress/fixtures/buzzTab/buzz.json");;
```

**Review by M-Abohasan — APPROVED** [review]
Ok.

**Pull request merged**